### DISCUSSION TITLE
Nach Update Fehler in Model Auswahl bei analog und digital in der Config

### DISCUSSION THREAD

**Discussion by Toki99** [jomjol/AI-on-the-edge-device#2025] [Q&A] [answered]
Hallo, nach einem Update bekomme ich den unten angefügten Fehler und kann auch keine neue Konfiguration des Bildes durchführen. Was kann ich tun? Danke!
![fehler](https://user-images.githubusercontent.com/49706480/218451289-34512f31-8574-469a-bb1f-5e65d74f3939.PNG)

**Comment by friedpa** [comment]
Steht bei Dir im Feld Model ein tflite Filename (welches auch in dem richtigen Verzeichnis liegt)?

![grafik](https://user-images.githubusercontent.com/42890695/218454975-e7e862e2-a854-4de9-98f7-4818c0646a88.png)

**Comment by Toki99** [comment]
Nein wie bekomme ich das File dort hin?

**Comment by Toki99** [comment, marked as answer]
Irgendwie sind bei mir die sourcen dureinander gekommen - jetzt habe ich das file
